lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -886,6 +904,35 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
     return std::distance(children.begin(), it);
   }
 
+private:
+  std::string mangledTypenameForTasksTuple(size_t count) {
+    /*
+    Global > TypeMangling > Type > Tuple
+      TupleElement > Type > Structure
+        Module, text="Swift"
+        Identifier, text="UnsafeCurrentTask"
+    */
+    using namespace ::swift::Demangle;
+    using Kind = Node::Kind;
+    NodeFactory factory;
+    auto [root, tuple] = swift_demangle::MakeNodeChain(
+        {Kind::TypeMangling, Kind::Type, Kind::Tuple}, factory);
+
+    // Make a TupleElement subtree N times, where N is the number of subtasks.
+    for (size_t i = 0; i < count; ++i) {
+      auto *structure = swift_demangle::MakeNodeChain(
+          tuple, {Kind::TupleElement, Kind::Type, Kind::Structure}, factory);
+      if (structure) {
+        structure->addChild(
+            factory.createNode(Kind::Module, ::swift::STDLIB_NAME), factory);
+        structure->addChild(
+            factory.createNode(Kind::Identifier, "UnsafeCurrentTask"), factory);
```

Comment:
In case you didn't already know, we also have 

```
  CompilerType
  CreateTupleType(const std::vector<TupleElement> &elements);
```
in TypeSystemSwift.

Reply:
I didn't, thanks.

Reply:
hmm, that interface isn't a match for this use case. I don't have a `CompilerType` instance for the element types. I could create one, but is it worth it to mangle the element type only for `CreateTupleType` to demangle it?

Reply:
Probably not.